test/libsolidity/syntaxTests/types/sized_type_variable.sol

```diff
@@ -0,0 +1,7 @@
+contract C {
+    function f() public pure {
+        uint bytes003200 = 0;
```

Comment:
Since after the change this becomes valid code, is this a bug fix or a breaking (bug) fix?

Reply:
Allowing something as identifier that was previously disallowed is non-breaking anyways, isn't it?